models/marts/core/int_all_graph_resources.sql

```diff
@@ -23,116 +23,116 @@ with unioned as (
 
 naming_convention_prefixes as (
     select * from {{ ref('stg_naming_convention_prefixes') }}
-), 
+),
 
 naming_convention_folders as (
     select * from {{ ref('stg_naming_convention_folders') }}
-), 
+),
 
 unioned_with_calc as (
-    select 
+    select
         *,
-        case 
+        case
             when resource_type = 'source' then  {{ dbt.concat(['source_name',"'.'",'name']) }}
-            when coalesce(version, '') != '' then {{ dbt.concat(['name',"'.v'",'version']) }} 
-            else name 
+            when coalesce(version, '') != '' then {{ dbt.concat(['name',"'.v'",'version']) }}
+            else name
         end as resource_name,
         case
             when resource_type = 'source' then null
-            else {{ dbt.split_part('name', "'_'", 1) }}||'_' 
+            else {{ dbt.split_part('name', "'_'", 1) }}||'_'
         end as prefix,
         {{ get_dbtreplace_directory_pattern() }} as directory_path,
         regexp_replace(file_path,'.*{{ get_regexp_directory_pattern() }}','') as file_name
     from unioned
     where coalesce(is_enabled, True) = True and package_name != 'dbt_project_evaluator'
-), 
+),
 
 joined as (
 
     select
-        unioned_with_calc.unique_id as resource_id, 
-        unioned_with_calc.resource_name, 
-        unioned_with_calc.prefix, 
-        unioned_with_calc.resource_type, 
-        unioned_with_calc.file_path, 
-        unioned_with_calc.directory_path,
-        unioned_with_calc.is_generic_test,
-        unioned_with_calc.file_name,
-        case 
+        unioned_with_calc.unique_id as resource_id,
+        unioned_with_calc.resource_name as resource_name,
```

Comment:
removed all unnecessary aliasing

Reply:
These are column names you get in older versions (i.e. Clickhouse v23.8).
<img width="835" alt="Screenshot 2024-07-31 at 07 36 08" src="https://github.com/user-attachments/assets/152c4382-c6cf-4f6c-9cf5-be57ac3dc78b">

Most of the downstream models fail because with similar errors because of this
```shell
04:31:28    Database Error in model fct_source_directories (models/marts/structure/fct_source_directories.sql)
  :HTTPDriver for http://localhost:8123 returned response code 404)
   Code: 47. DB::Exception: Missing columns: 'file_name' 'source_name' 'file_path' 'directory_path' while processing query: 'SELECT resource_name, resource_type, file_path AS current_file_path, concat('models/', 'staging', '/', source_name, '/', file_name) AS change_file_path_to FROM all_graph_resources WHERE (resource_type = 'source') AND (directory_path NOT LIKE concat('%', source_name, '%'))', required columns: 'resource_name' 'directory_path' 'resource_type' 'file_path' 'source_name' 'file_name' 'resource_name' 'directory_path' 'resource_type' 'file_path' 'source_name' 'file_name'. (UNKNOWN_IDENTIFIER) (version 23.8.16.40 (official build))
  compiled Code at target/run/dbt_project_evaluator/models/marts/structure/fct_source_directories.sql
```